setup.cfg

```diff
@@ -185,23 +185,95 @@ known_third_party = atlas4py,attr,black,boltons,click,dace,devtools,factory,hypo
 
 #-- mypy --
 [mypy]
-follow_imports = silent
-ignore_missing_imports = True
+# Global options
+install_types = True
+non_interactive = True
+
+# File paths and patterns
+exclude = (?x)(
+          /_external_src/
+          | /build/
+          | /setup\.py$
+          | /tests/
```

Comment:
Does it mean tests are generally excluded?

Reply:
Yes, they are fully excluded, this setting was already in place, but maybe we should change it in the future...